src/gengowatcher/config_with_autoaccept.py

```
@@ -0,0 +1,198 @@
+            "browser_profile_path": "",
+            "notification_on_accept": True,
+            "log_acceptance": True,
+        },
+        "Captcha": {
+            "service": "",  # "2captcha" or "anti-captcha"
+            "api_key": "",
+            "max_retries": 3,
+            "retry_delay": 5,
+            "rate_limit": 60,  # requests per minute
+        },
+    }
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)